### PR TITLE
Fix tune

### LINKED ISSUE #303: TypeError: 'EnumTypeWrapper' object is not callable when trying to play a tune
Hi,
I'm trying to play a tone on my system but I'm getting this error (TypeError: 'EnumTypeWrapper').  As per the TuneDescription and SongElement documentation, I've made the below code. Can anybody let me know why the below code isn't working.

```
#!/usr/bin/env python3

import asyncio
from mavsdk import System
from mavsdk.tune import (TuneDescription,SongElement,TuneError)
import traceback


async def run():

    drone = System(mavsdk_server_address="localhost", port=50041)
    await drone.connect()

    print("Waiting for drone to connect...")
    async for state in drone.core.connection_state():
        if state.is_connected:
            print(f"Drone discovered with UUID: {state.uuid}")
            break

    id_info = await drone.info.get_identification()
    print(id_info)
    await asyncio.sleep(1)

    song_elem = []
    song_elem.append(SongElement.NOTE_G)
    try:
        tone1 = TuneDescription(song_elem,100)
        await drone.tune.play_tune(tone1)
    except Exception:
        print(traceback.print_exc())

    print("Tone played")


if __name__ == "__main__":
    loop = asyncio.get_event_loop()
    loop.run_until_complete(run())
```

Output:
```
Waiting for mavsdk_server to be ready...
Connected to mavsdk_server!
Waiting for drone to connect...
Drone discovered with UUID: 0
Identification: [hardware_uid: 000200000000353236353038510a001c002c]
Traceback (most recent call last):
  File "play_tone.py", line 28, in run
    await drone.tune.play_tune(tone1)
  File "/home/aifi/.local/lib/python3.8/site-packages/mavsdk/tune.py", line 460, in play_tune
    tune_description.translate_to_rpc(request.tune_description)
  File "/home/aifi/.local/lib/python3.8/site-packages/mavsdk/tune.py", line 261, in translate_to_rpc
    rpc_elem = tune_pb2.SongElement()
TypeError: 'EnumTypeWrapper' object is not callable
None
Tone played
```

### PR DESCRIPTION
**Pull request by JonasVautherin**
Repeated enums were not translated correctly.

Took the opportunity to add an example (autumn leaves, again :notes:)

Resolves #303

### CODE REVIEW

File: mavsdk/tune.py [hunk 1]
```diff
@@ -258,9 +258,10 @@ def translate_to_rpc(self, rpcTuneDescription):
             
         rpc_elems_list = []
         for elem in self.song_elements:
-            rpc_elem = tune_pb2.SongElement()
-            elem.translate_to_rpc(rpc_elem)
-            rpc_elems_list.append(rpc_elem)
+                
+            rpc_elem = tune_pb2.SongElement
+            rpc_elems_list.append(elem.translate_to_rpc())
```
Comment (julianoes) [inline mavsdk/tune.py:263]: This doesn't look correct, does it?

Comment (JonasVautherin) [reply mavsdk/tune.py:263]: oh, I don't know why I kept `rpc_elem` at all :see_no_evil:. My bad!

Comment (JonasVautherin) [reply mavsdk/tune.py:262]: Should be better now :+1:

**Comment by JonasVautherin** [comment]
Found a buzzer and managed to test it with Pixhawk 1! It works :blush:

**Comment by julianoes** [comment]
Awesome!

**Review by julianoes — APPROVED** [review]

**Pull request merged**